docs/docs/pixabay-source-plugin-tutorial.md

```diff
@@ -28,15 +28,15 @@ _**NOTE:** if your data is local i.e. on your file system and part of your site'
 
 Your plugin is going to source images from Pixabay. You'll be able to configure your plugin in your site's `gatsby-config.js` file and write GraphQL queries to access your plugin's data.
 
-> **NOTE:** You'll need a Pixabay API key which you can get by [registering for a Pixabay account](https://pixabay.com/en/accounts/register/). Your API key is in the [“Search Images” section of the Pixabay API docs](https://pixabay.com/api/docs/#api_search_images).
+> **NOTE:** You'll need a Pixabay API key which you can get by [registering for a Pixabay account](https://pixabay.com/en/accounts/register/). Your API key is in the ["Search Images" section of the Pixabay API docs](https://pixabay.com/api/docs/#api_search_images).
```

Comment:
Curly quotes aren't necessary to change – in fact they are more appropriate for bodies of text. Was there a specific reason to change this?

Reply:
no specific reason - i just thought they are not "the right ones" ^^ because they look a little bit "at an angle"